README.md

```diff
@@ -237,6 +237,10 @@ kubectl get pods -n netobserv -l app=netobserv-plugin
 kubectl logs -n netobserv -l app=netobserv-plugin
 ```
 
+### I first deployed flowcollector, and then kafka. Flowlogs-pipeline is not consuming any flow from Kafka
+
+This is a known bug in one of flowlogs-pipeline dependencies. Please recreate the flowlogs-pipeline pods by killing them, they will be automatically recreated.
```

Comment:
Maybe add the link to the issue in the kafka lib, if it exists. I'd maybe suggest, as another alternative, to redeploy the flow collector, in addition to killing pods one by one.